foia_hub/management/commands/load_agency_contacts.py

```diff
@@ -22,9 +15,33 @@ class Command(BaseCommand):
     """
 
     def handle(self, *args, **options):
-        try:
+        if len(args) > 0:
             yaml_folder = args[0]
-        except IndexError:
-            yaml_folder = _get_yaml_folder()
+        else:
+            yaml_folder = download_data()
 
+        print("Data directory: %s" % yaml_folder)
+        print("Loading data...")
         process_yamls(yaml_folder)
+
+# Clone a new copy of the git repo, removing the dir if it exists.
+def download_data():
+    print("No directory given, checking out copy of data from version control.")
+    print("Note: this requires git to be installed.")
+
+    path = temp_home()
+    if os.path.exists(path):
+        print("Deleting old data checkout.")
+        shutil.rmtree(path)
+
+    print("Cloning repository at %s" % DEFAULT_DATA_REPO)
+    clone_repo(DEFAULT_DATA_REPO, path)
+
+    return os.path.join(path, "contacts", "data")
+
+def temp_home():
+    return os.path.join(BASE_DIR, "..", "temp-data")
+
+# test if a command exists, don't print output
```

Comment:
Whoops, yeah, that's cruft from some code I adapted this line from. Removed in 4f99865.